### DISCUSSION TITLE
Offline API Docs?

### DISCUSSION THREAD

**Discussion by mikey-t** [nginx/agent#402] [Q&A] [answered]
Is there an "offline" way to view the REST API docs without installing nginx, nginx-agent, go and enabling swagger? I'd like to get a feel for what can and can't be done via the REST API before I decide whether I want to install and use it.

**Comment by dhurley** [comment, marked as answer]
Hi @mikey-t, the REST API definitions are defined here in this swagger.json file https://github.com/nginx/agent/blob/main/docs/swagger.json. You can use a swagger editor like this one https://editor-next.swagger.io/ to view it more easily. Just copy the contents of the json file into the editor and it will display a preview of the docs.

**Reply by mikey-t** [reply]
That's what I needed, thanks!
